sbt-bridge/src/xsbt/Problem.java

```diff
@@ -0,0 +1,45 @@
+package xsbt;
+
+import java.util.Optional;
+import xsbti.Position;
+import xsbti.Severity;
+
+final public class Problem implements xsbti.Problem {
```

Comment:
I am wondering if you need to implement `equals` and `hashCode` since it was a case case

Reply:
no, it was just a case class for convenience